NServiceBus.FileBasedRouting/FileBasedRoutingConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NServiceBus.Configuration.AdvanceExtensibility;
+
+namespace NServiceBus.FileBasedRouting
+{
+    public static class FileBasedRoutingConfig
+    {
+        /// <summary>
+        /// Enables routing configured with the routing configuration file.
+        /// </summary>
+        /// <param name="config">The configuration object.</param>
+        public static void EnableFileBasedRouting(this EndpointConfiguration config)
```

Comment:
Should we throw an exception if somebody wants to use centralized routing for events with a transport that supports native event routing?